Emby.Server.Implementations/Plugins/PluginManager.cs

```diff
@@ -514,12 +514,10 @@ private LocalPlugin LoadManifest(string dir)
             {
                 try
                 {
-                    var data = File.ReadAllText(metafile, Encoding.UTF8);
+                    var data = File.ReadAllBytes(metafile);
                     manifest = JsonSerializer.Deserialize<PluginManifest>(data, _jsonOptions);
                 }
-#pragma warning disable CA1031 // Do not catch general exception types
-                catch (Exception ex)
-#pragma warning restore CA1031 // Do not catch general exception types
+                catch (JsonException ex)
```

Comment:
I don't think this should be changed, we don't want IO errors to kill all plugin processing.